microsoft-365/security/defender-endpoint/android-configure.md

```diff
@@ -298,18 +298,18 @@ Use the following steps to turn on privacy for targeted users in the work profil
 
 Using this privacy control won't impact the device compliance check or conditional access. For example, devices with a malicious app will always have a risk level of "Medium".
 
-## Disable sign out
-Defender for Endpoint supports deployment without sign out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering the device.
+## Disable sign-out
 
-Use the following steps to configure the Disable sign out:
+Defender for Endpoint supports deployment without sign-out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering the device.
+Use the following steps to configure Disable sign-out:
```

Comment:
```suggestion
Defender for Endpoint supports deployment without the sign-out button in the app to prevent users from signing out of the Defender app. This is important to prevent users from tampering with the device.
Use the following steps to configure Disable sign-out:
```